sousmot/sousmotapp/templates/registration/signup.html

```diff
@@ -0,0 +1,73 @@
+{% extends "sousmotapp/base.html" %}
+
+{% block title %}Sign Up{% endblock %}
+
+{% block content %}
+
+    <section class="hero has-text-centered">
+        <div class="hero-body">
+            <h1 class="title">Sign Up</h1>
+            <p class="subtitle">Nice to meet you, join-us !</p>
+
+            <div class="box">
+                <form method="post">
+                    {% csrf_token %}
+                    <!-- Username -->
+                    <div class="field">
+                        <label for="id_username" class="label">Username</label>
+                        <div class="control">
+                            <input id="id_username" type="text" name="username" class="input" required>
+                        </div>
+                    </div>
+                    <!-- Password 1 -->
+                    <div class="field">
+                        <label for="id_password1" class="label">Password</label>
+                        <div class="control">
+                            <input id="id_password1" type="password" name="password1" class="input" required>
+                        </div>
+                    </div>
+                    <!-- Password 2 -->
+                    <div class="field">
+                        <label for="id_password2" class="label">Confirm Password</label>
+                        <div class="control">
+                            <input id="id_password2" type="password" name="password2" class="input"
+                                   oninput="matchPassword()" required>
+                        </div>
+                    </div>
+                    <div class="notification is-info is-light">
+                        <ul>
+                            <li>Your password can’t be too similar to your other personal information.</li>
+                            <li>Your password must contain at least 8 characters.</li>
+                            <li>Your password can’t be a commonly used password.</li>
+                            <li>Your password can’t be entirely numeric.</li>
+                        </ul>
+                    </div>
```

Comment:
Are you sure that the info about passwords should be always visible ? :thinking: